.github/workflows/module_integration.yml

```
@@ -133,3 +133,73 @@ jobs:
           . .venv/bin/activate
           ./cmake_build/test/test_routing_pybind
           deactivate
+  
+  test_forcing_engine_integration:
+    # TODO: Remove when integration is ready
+    if: false
+
+    name: Test Forcings Engine Integration
+    runs-on: 'ubuntu-latest'
+    env:
+      CC: mpicc
+      CXX: mpic++
+      FC: gfortran
+      F90: gfortran
+
+    steps:
+      - name: Checkout NGen
+        uses: actions/checkout@v4
+      
+      - name: Checkout Forcings Engine
+        uses: actions/checkout@v4
+        with:
+          # TODO: replace once merged, https://github.com/NOAA-OWP/ngen-forcing/pull/9
+          # repository: NOAA-OWP/ngen-forcing
+          repository: jduckerOWP/ngen-forcing
```

```suggestion
          repository: NOAA-OWP/ngen-forcing
```
Referenced PR was merged.